sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputRequireFailForTryCatchInJunitCheck11.java

```diff
@@ -51,4 +51,52 @@ public void violation() {
             assertThat(expected).isInstanceOf(NullPointerException.class);
         }
     }
+
+    @Test
+    public void validAssertWithMessage() {
+        try {
+            obj.toString();
+            assertWithMessage("simple message").fail();
```

Comment:
Can you please add some cases that will produce a violation, like `assertWithMessage` without any `fail` call. The fail is what is really required for this check.

Reply:
done